### ISSUE TITLE
CFP: mydefichain Infrastructure 2025 (2,625,000 DFI)

### ISSUE THREAD

**Issue by Stonygan** [Stonygan/powershell#10]
This funding is intended to ensure the continuation of the mydefichain infrastructure in 2025.
There is only one CFP for our infrastructure, as we have merged various services to save costs without affecting performance and stability.

### What does the funding involve?

**The mydefichain Ocean: https://ocean.mydefichain.com/**
   - for the LightWallet, Vault-Maxi, BNB-Bridge (defichain-bridge.com), API services...

![image](https://github.com/user-attachments/assets/ec6b21fa-4e50-417b-bf40-e4f907b61590)

**RPC Endpoint: https://dmc.mydefichain.com/**
   - for the LightWallet, MetaMask, Blockscout, other EVM services
   - for Mainnet, testnet3 and partly changi.

![image](https://github.com/user-attachments/assets/9122b274-0e2b-4fa3-ad7b-4a1bdf992250)

![image](https://github.com/user-attachments/assets/65ea8358-e0d7-48af-96c5-0f82b8aafa1e)

**Various block explorers**
  - Native explorer based on Ocean: https://explorer.mydefichain.com/
  - EVM-Explorer Blockscout mainnet: https://mainnet-dmc.mydefichain.com/
  - EVM Explorer Blockscout mainnet old version: https://mainnet-dmc.mydefichain.com:8441/ (for some exchanges such as Bitrue)
  - EVM-Explorer Blockscout testnet: https://testnet-dmc.mydefichain.com/
  - EVM-Explorer Blockscout changi: https://changi-dmc.mydefichain.com/

**Miscellaneous**
- API services for defichain-analytics.com, for example
   - Mainnet: https://api.mydefichain.com/v1/
   - Testnet : http://tn01.mydefichain.com/
   - changi: http://cw01.mydefichain.com/api/
- RPC bookmark and status page: https://rpc.mydefichain.com
- Defichain Statuspage: https://www.defichain-status.com

**Snapshot Service**

- Snapshot Mainnet: https://snapshot-de.mydefichain.com
- Snapshot Testnet: https://testnet.snapshot-de.mydefichain.com
- Snapshot Ocean: https://ocean-snapshot.mydefichain.com

**Faucet**

- Mainnet: https://utxo.mydefichain.com/
- Testnet: https://testnet-utxo.mydefichain.com/
- changi: https://changi-utxo.mydefichain.com/faucet

  
**Seeder**
- Mainnet: seed.mydeficha.in (included in the official Node-Version, Statuspage: https://status.defichain-masternode.com)
- Testnet: seeder-testnet3.defichain-status.com (with Network Crawler)

### Statement of costs:

- changi RPC:	                           $0,00
- testnet RPC:	                           $0,00
- mainnet RPC:	                   $2.000,00
- Ocean:		                           $1.000,00
- Snapshot, Faucet, API, Seeder: $500 
- Personnel:	        see seperated CFP

We are discontinuing support for changi on a large scale and will only operate the nodes when required. The 2024 costs of $250 per month will no longer be charged.
We have reduced the testnet infrastructure by 50% and are also using some of the mainnet nodes for this, so that there are also no more costs for the testnet, the $500 for this is saved.
We continue to operate the RPC endpoints for mainnet and testnet as before, as these are also used massively by LightWallet and Blockexplorer and requests have increased more than a factor of ten compared to June 2024.

In summary, the costs for the infrastructure RPC and Ocean amount to $3500 per month, which means $42,000 for the year 2025 and at the current exchange rate of $0.016/DFI 2,625,000 DFI, which we are requesting via the Community Fund.
